cypress/support/commands/login.ts

```diff
@@ -29,7 +29,8 @@ export const login = (username: string, password: string) => {
   //   });
   // });
   cy.get('button[id="btnSignIn"]').should("exist").click();
-  cy.wait(6000);
+  // Something is causing Next -> Keycloak load time to be long
+  cy.wait(12000);
```

Comment:
good catch, but this wasn't previously the case (around the time of the SQL>POSTGRES move) 

unfortunately, the more tests we add, the more these timeouts will compound, adding several minutes to the total test run-time.

Reply:
The thing is that SQL -> Postgres shouldn't be causing this.
Keycloak was _always_ on Postgres.
As far as I can replicate, this only happens via Cypress (Chromium and Electron).
If you spin it up on your local, it's quick.